golang/arguments.go

```diff
@@ -0,0 +1,103 @@
+package tchannel
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+)
+
+// An Input is able to read an argument from a call body
```

Comment:
This is already golint-compatible (comments for types can start with an article).  But a bunch of other stuff is not (such as the lack of godoc comments on the ReadFrom/WriteTo implementations).  Will update.